pyrcareworld/pyrcareworld/envs/base_env.py

```diff
@@ -16,7 +16,7 @@ class RCareWorld(ABC):
 
     Args:
         executable_file: Str, the absolute path of Unity executable file. None for last used executable file; "@editor" for using Unity Editor.
-        scene_file: Str, the absolute path of Unity scene JSON file. All JSON files locate at `StraemingAssets/SceneData` by default.
+        scene_file: Str, the absolute path of Unity scene JSON file. All JSON files locate at `StreamingAssets/SceneData` by default.
```

Comment:
Is this comment up to date? I don't see a `StreamingAssets` directory

Reply:
This refers to the Unity project's directory. Needs to be clarified somewhere. I'll make a saperate PR to fix it.